### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by KapJI**
- Remove duplicating license section.
- Fix typo.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Pull requests are the best way to propose changes to the codebase.
 
 1. Fork the repo and create your branch from `master`.
 2. If you've changed something, update the documentation.
-3. Make sure your code lints (using pre-commit).
+3. Make sure your code lints (using `pre-commit`).
```
Comment (DurgNomis-drol) [inline CONTRIBUTING.md:18]: Shouldn't it say  "Make sure your code lints passes"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -91,7 +91,3 @@ You can run the tests on all repository file with the command:
 ```console
 $ poetry run pre-commit run --all-files
 ```
-
-## License
```
Comment (DurgNomis-drol) [inline CONTRIBUTING.md:95]: Why should we remove this section? Is it a good thing to explicit state this?

Comment (KapJI) [reply CONTRIBUTING.md:95]: There is already a section about this: https://github.com/leikoilja/ha-google-home/blob/master/CONTRIBUTING.md#any-contributions-you-make-will-be-under-the-mit-software-license

File: CONTRIBUTING.md (hunk 1 above)
Comment (KapJI) [reply CONTRIBUTING.md]: Updated this.

File: CONTRIBUTING.md (hunk 2 above)
Comment (DurgNomis-drol) [reply CONTRIBUTING.md:95]: Didn't see that 😊

**Review by DurgNomis-drol — APPROVED** [review]
LGTM! Great job!

**Pull request merged**